python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -357,30 +354,29 @@ def parse_node_data(file_idx, in_file, feat_ops, node_id_col, label_conf,
         The configuration of labels.
     read_file : callable
         The function to read the node file
-    return_dict : dict
-        The dictionary that is shared among all processes and saves the parsed node data.
+
+    Returns
+    -------
+    tuple : node ID array and a dict of node feature tensors.
     """
     data = read_file(in_file)
     feat_data = process_features(data, feat_ops) if feat_ops is not None else {}
     if label_conf is not None:
         label_data = process_labels(data, label_conf)
         for key, val in label_data.items():
             feat_data[key] = val
-    return_dict[file_idx] = (data[node_id_col], feat_data)
+    return (data[node_id_col], feat_data)
 
-def parse_edge_data(file_idx, in_file, feat_ops, src_id_col, dst_id_col, edge_type,
-                    node_id_map, label_conf, read_file, return_dict):
+def parse_edge_data(in_file, feat_ops, src_id_col, dst_id_col, edge_type,
+                    node_id_map, label_conf, read_file):
     """ Parse edge data.
 
     The function parses an edge file that contains the source and destination node
     IDs, edge features and potentially edge labels. The edge file is parsed
-    according to users' configuration and performs some feature transformation
-    and save the result in `return_dict`.
+    according to users' configuration and performs some feature transformation.
```

Comment:
Can we tell what kinds of feature transformation GFS supports?

Reply:
This may not be the best place to document the transformation the pipeline supports. We need to have proper documentation for the pipeline.